src/components/flatList.tsx

```diff
@@ -1,36 +1,75 @@
 import React from 'react';
 import { ListRenderItem } from 'react-native';
 import { SafeAreaView } from 'react-native';
-import { StyleSheet, View, FlatList, Text, StatusBar } from 'react-native';
+import { StyleSheet, View, FlatList, Text } from 'react-native';
 import { Journey } from '../model/fareResponseClass';
 
 const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    marginTop: StatusBar.currentHeight || 0,
-  },
   item: {
-    backgroundColor: '#f9c2ff',
+    backgroundColor: '#F2F2F2',
     padding: 10,
-    marginVertical: 2,
-    marginHorizontal: 4,
+    marginVertical: 8,
+    marginHorizontal: 10,
+    minWidth: '90%',
   },
-  title: {
+  body: {
     fontSize: 14,
+    color: '#000',
+  },
+  heading: {
+    fontSize: 16,
+    fontWeight: '600',
+    color: '#000',
+  },
+  columnContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
   },
 });
 
 type FlatListProps = {
   items: Journey[];
 };
 
+const convertDateString = (isoDateString: string): string => {
```

Comment:
I think these method names can be a bit clearer, maybe `convertIsoDateTimeToFormattedDate` and `convertIsoDateTimeToFormattedTime`